packages/browserid-verifier/tests/lib/verifier.js

```diff
@@ -136,10 +136,10 @@ Verifier.prototype.stop = function (cb) {
   if (!this.process || !this._url) {
     throw new Error('verifier not running');
   }
-  this.process.kill('SIGINT');
   this.process.on('exit', function (code) {
     cb(!code ? null : 'non-zero exit code: ' + code);
   });
+  this.process.kill('SIGINT');
```

Comment:
Killing the child process before the listener was attached is flakey because a fast enough system will kill it and trigger the event before the new listener is registered.